lazyflow/utility/roiRequestBatch.py

```diff
@@ -247,7 +250,7 @@ def _handleCompletedRequest(self, roi, result):
                 # Report progress (if possible)
                 if self._totalVolume is not None:
                     self._processedVolume += numpy.prod(numpy.subtract(roi[1], roi[0]))
```

Comment:
Shouldn't the same problem happen here with `numpy.prod`?

Reply:
good catch - that actually means that I should rethink the fix, as this only enforces `_totalVolume` to be `int` once set in this class. It is, however, calculated itself outside this class with `numpy.prod`...

Reply:
you got me paranoid ;)